e33lin/grocery-store
# Form for number of stores working

**Summary**
- created a form on the create list page using radio buttons for user to indicate whether they want to shop at 1 store or 2 stores
- radio buttons only show when user has something in their list 
- radio buttons default to 1

**Testing**
- go to /lists and verify that when nothing is in your list, you can't see the radio buttons
- add something to list and verify radio buttons are there and 1 store is selected by default
- click "see recommendations", should take you to /recommendations
- go to terminal and make sure n_stores matches the button you selected 
<img width="716" alt="image" src="https://user-images.githubusercontent.com/65199704/218559241-6e2ef26b-771a-420e-99c6-b0b431a3b3fa.png">
